File: app/src/main/java/de/mchllngr/quickopen/model/RemovedApplicationModel.java
package de.mchllngr.quickopen.model;

/**
 * Class for holding the needed data about an removed item for the remove to be undone.
 */
public class RemovedApplicationModel {

    /**
     * Last position of removed item.
     */
    public int position;
    /**
     * The removed item.
     */
    public ApplicationModel applicationModel;

    /**
     * Constructor for creating the {@link RemovedApplicationModel}.
     */
    public RemovedApplicationModel(int position, ApplicationModel applicationModel) {
        this.position = position;
        this.applicationModel = applicationModel;
    }
}

File: app/src/main/java/de/mchllngr/quickero/model/RemovedApplicationModel.kt
package de.mchllngr.quickero.model

/**
 * Class for holding the needed data about an removed item for the remove to be undone.
 */
/**
 * Constructor for creating the [RemovedApplicationModel].
 */
class RemovedApplicationModel(
    /**
     * Last position of removed item.
     */
    val position: Int,
    /**
     * The removed item.
     */
    val applicationModel: ApplicationModel
)

